### DISCUSSION TITLE
Support spaceBeforeFunctionParens formatting rule

### DISCUSSION THREAD

**Discussion by galvez** [biomejs/biome#1487] [Ideas]
In [StandardJS](https://standardjs.com/rules), there's always a space before a function's parenthesis:

```diff
- function foobar() {
+ function foobar () {
```

The main reasoning I think it's that it makes instantly easier to search for a function definition by just adopting the convention that no space means it's a call, space means it's a definition.

Overall, it would be nice to be able to fully replicate StandardJS with Biome.

**Comment by galvez** [comment]
My apologies, I see that this is a duplicate of https://github.com/biomejs/biome/discussions/1193 — still I think this makes for a worthwhile addition, despite the acceptedly rational draconian approach of maintaining the rule set minimal.

**Comment by Conaclos** [comment]
Closing because of duplication.
